incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/config/OtlpOverridesConfig.java

```diff
@@ -12,31 +12,23 @@
  * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package io.aklivity.zilla.runtime.command.metrics.internal.utils;
+package io.aklivity.zilla.runtime.exporter.otlp.internal.config;
 
-import io.aklivity.zilla.runtime.engine.EngineContext;
-import io.aklivity.zilla.runtime.engine.metrics.MetricContext;
-public interface Metric
+import io.aklivity.zilla.runtime.engine.config.OptionsConfig;
+
+public class OtlpOverridesConfig extends OptionsConfig
 {
-    enum Kind
-    {
-        COUNTER,
-        GAUGE,
-        HISTOGRAM
-    }
+    public String metrics;
+    public String logs;
+    public String traces;
```

Comment:
These should be of type URI instead of String, but not yet resolved against the base, so you can round-trip the config back to JSON accurately.

Reply:
fixed